src/pages/Route/components/Step2/RequestRewriteView.tsx

```diff
@@ -29,7 +29,7 @@ const RequestRewriteView: React.FC<RouteModule.Step2PassProps> = ({
     fetchUpstreamList().then(({ data }) => setList(data));
   }, []);
   return (
-    <UpstreamForm ref={upstreamRef} form={form} disabled={disabled} list={list} showSelector />
+    <UpstreamForm ref={upstreamRef} form={form} disabled={disabled} list={list} showSelector key={1} />
```

Comment:
Why we need this key?

Reply:
this key uses for fix route step2 warnings.